cdap-ui/app/cdap/components/Footer/index.tsx

```diff
@@ -20,7 +20,48 @@ import If from '../If';
 import { isNilOrEmptyString, objectQuery } from 'services/helpers';
 import { getCurrentNamespace } from 'services/NamespaceStore';
 import NamespaceStore from 'services/NamespaceStore';
-require('./Footer.scss');
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    background: 'white',
+    color: '#cccccc',
```

Comment:
The colors are available from theme. Eg: https://github.com/cdapio/cdap/blob/3518f3bd0e9b60860722868bc45315d0087f963b/cdap-ui/app/cdap/components/LogViewer/TopPanel.tsx#L36